src/test/adapter-factory/uuid.ts

```
@@ -0,0 +1,68 @@
+					data: {
+						...user,
+						//@ts-expect-error - remove id from `user`
+						id: undefined,
+					},
+				});
+				expect(res).toHaveProperty("id");
+				expect(typeof res.id).toBe("string");
+				const uuidRegex =
+					/^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i;
+				expect(res.id).toMatch(uuidRegex);
+				console.log(res);
```

ESLint

[error] Unexpected console statement.

(no-console)